Sources/ArgumentParser/Completions/CompletionsGenerator.swift

```diff
@@ -42,10 +42,14 @@ public struct CompletionShell: RawRepresentable, Hashable, CaseIterable {
 
   /// Returns an instance representing the current shell, if recognized.
   public static func autodetected() -> CompletionShell? {
+#if os(Windows)
+    return nil
+#elseif
```

Comment:
Looks like this should just be an `else`:
```suggestion
#else
```